cves/kernel/CVE-2015-6526.yml

```diff
@@ -266,8 +294,11 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
```

Comment:
I'm confused by this note here and whether or not a privileged user can take advantage of the vulnerability and what you mean by no one should.

Reply:
Good point this wording is confusing.